lib/cadet/assessments/assessments.ex

```diff
@@ -112,32 +109,28 @@ defmodule Cadet.Assessments do
     end
   end
 
-  def user_with_group(%User{id: id}) do
-    User
-    |> preload(:group)
-    |> Repo.get(id)
-  end
-
-  def user_current_story(user = %User{}) do
+  # :TODO to check how this story works
```

Comment:
Basically gets the story of the latest attempted or unattempted assessment with a non-empty `story` field